python/ray/serve/handle.py

```diff
@@ -162,6 +163,18 @@ def _get_or_create_router(self) -> Router:
             except Exception:
                 availability_zone = None
 
+            if not self.__class__ == DeploymentHandle:
+                warnings.warn(
```

Comment:
@akshay-anyscale @shrekris-anyscale added this explicit warning message PTAL

Reply:
Should we explicitly call out the other 2 APIs as deprecated?

Reply:
done